x-pack/plugins/security/server/authentication/providers/saml.ts

```diff
@@ -19,15 +19,11 @@ import { AuthenticationProviderOptions, BaseAuthenticationProvider } from './bas
  * The state supported by the provider (for the SAML handshake or established session).
  */
 interface ProviderState extends Partial<TokenPair> {
-  /**
-   * Username of the SAML authenticated user.
-   */
-  username?: string;
```

Comment:
__note:__ Now Authenticator decides whether or not user should be redirected to Overwritten Session UI.